src/components/ui/ExtraSmallText.tsx

```diff
@@ -0,0 +1,22 @@
+import { Trans } from '@lingui/macro';
+import React from 'react';
+
+type ExtraSmallTextProps = {
+    color: string,
+    children: any;
+    extendedClassNames: any;
+}
+
+const ExtraSmallText: React.FC<ExtraSmallTextProps> = ({ color, children, extendedClassNames }) => {
```

Comment:
This component doesnt seem to be used anywhere

Reply:
Yeah, I will delete that.